tests/validators/invalid_urls.txt

```diff
@@ -50,3 +50,4 @@ http://[::1:2::3]:8080/
 http://[]
 http://[]:8080
 http://example..com/
+://example.com
```

Comment:
Makes sense, could you modify the test then? I don't think we need to backport it to 1.8.